.gitignore

```diff
@@ -1,5 +1,6 @@
 /.build
 /.release
 /.tarballs
+.idea/
```

Comment:
I'd rather not have editor-specific settings in here. You can ignore .idea for yourself in `.git/info/exclude` (this repository only) or [create a global gitignore file](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) for yourself.